volunteer/index.html

```diff
@@ -2,12 +2,40 @@
 enable-mentor-alert: false
 title: "Volunteer Application Form"
 scripts: |
-    <script src="js/spring15android.js"></script>
 ---
+<div class="banner-image volunteer">
+    <h1 class="banner-title">Volunteers</h1>
+</div>
+
+<!--Card-->
+<div class="row">
+  <div class="col-sm-6 col-md-6">
+    <div class="thumbnail volunteer-card" style="margin-right: 5%;">
+      <img src="../images/volunteers/classroomVolunteers.jpg">
+        <div class="caption">
+          <h3>Classroom Volunteer</h3>
+          <p>Classroom Volunteers play an essential role by providing one on one support to our students and assisting our instructors. With an understanding of the fundamentals of HTML, CSS, java, and javascript, you will engage a diverse group of students through a project based coding course. </p>
+        </div>
+    </div>
+  </div>
+
+  <div class="col-sm-6 col-md-6">
+    <div class="thumbnail volunteer-card" style="margin-left: 5%;">
```

Comment:
if you follow what I recommended in the previous comment, don't forget to remove this inline CSS.

Reply:
@AbelRR don't forget to remove this inline css (style="margin-left: 5%;) Right now the 2 cards have different widths.